### PR TITLE
add discussion question for linux

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/linux/user-and-file-management/linux-intro/linux-common-terminology.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/linux/user-and-file-management/linux-intro/linux-common-terminology.md?ref=acc2ac6bae1f18c4f8a31e3416d2f722b0d10683)

File: linux/user-and-file-management/linux-intro/linux-common-terminology.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ The **package manager** removes the tedious process of installing or uninstallin
 
 > 💡 As with all the skills in the app, if you want to access more advanced or specific content at any time, **feel free to use the *Skill Map* tab** to jump to the workouts most relevant for you.
 
+> 💬 Why are you interested in learning Linux?
```
Comment (kapnobatai137) [inline linux/user-and-file-management/linux-intro/linux-common-terminology.md:29]: this sounds a bit weird to me, maybe we can say "learning about Linux"?

cc @nem035 @Loopiezlol 

```suggestion
> 💬 Why are you interested in learning about Linux?
```

Comment (Loopiezlol) [reply linux/user-and-file-management/linux-intro/linux-common-terminology.md:29]: totally agree

**Review by Loopiezlol — CHANGES_REQUESTED** [review]
.

**Pull request merged**